src/Chapter1.hs

```diff
@@ -631,9 +641,14 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 = positiveSumLast2 . abs
+  where
+    positiveSumLast2 :: Int -> Int
+    positiveSumLast2 n = last + secondLast
+      where
+        last = n `mod` 10
```

Comment:
`last` is the name of a standard function. Haskell allows name shadowing, that's why it works. Though, in practice, it's usually better to avoid name clashes with other functions cause it could be confusing 🙂